manifests/4.4/cluster-loggings.crd.yaml

```diff
@@ -118,6 +118,62 @@ spec:
                       - "MultipleRedundancy"
                       - "SingleRedundancy"
                       - "ZeroRedundancy"
+                    indexManagement:
```

Comment:
The cluster logging CRD should define `retentionPolicy` as spec'd here: https://github.com/openshift/enhancements/blob/master/enhancements/cluster-logging/cluster-logging-es-rollover-data-design.md#clusterlogging-api  No need to provide `indexManagement` which is consumed by the CLO and defined by the EO as spec'd here https://github.com/openshift/enhancements/blob/master/enhancements/cluster-logging/cluster-logging-es-rollover-data-design.md#elasticsearch-api